### PR TITLE
Change Temp Current Sequence No to BigInteger

### PR DESCRIPTION
**Pull request by haoranpb**
#### Summary 

Replace `field(15; "Temp Current Sequence No."; Integer)` with `BigInteger` type.

Directly updating the field with new capability from Platform.

#### Work Item(s) 
Fixes [AB#535564](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/535564)

### CODE REVIEW

File: src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al [hunk 1]
```diff
@@ -131,6 +131,16 @@ table 309 "No. Series Line"
...
             DataClassification = SystemMetadata;
             Access = Internal;
+            ObsoleteReason = 'This field is replaced by field 16 - "Temp. Current Sequence No.".';
+            ObsoleteTag = '24.0';
+            ObsoleteState = Removed;
+        }
+        field(16; "Temp. Current Sequence No."; BigInteger)
+        {
+            Caption = 'Temporary Sequence Number';
+            Description = 'The field should only have value during runtime';
```
Comment (AndreasMoth) [inline src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al]: Maybe "This field holds the next sequence number during batching for sequences. It is only used during runtime and no values will be stored in the database.

**Review by AndreasMoth — DISMISSED** [review]

File: src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al (hunk 1 above)
Comment (haoranpb) [reply src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al]:
```suggestion
            Description = 'This field holds the next sequence number during batching for sequences. It is only used during runtime and no values will be stored in the database.';
```

**Review by grobyns — APPROVED** [review]

File: src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al [hunk 2]
```diff
@@ -131,6 +131,18 @@ table 309 "No. Series Line"
             Caption = 'Temporary Sequence Number';
             DataClassification = SystemMetadata;
             Access = Internal;
+            ObsoleteReason = 'This field is replaced by field 16 - "Temp. Current Sequence No.".';
+#pragma warning disable AS0115
+            ObsoleteState = Removed;
```
Comment (StefanSosic) [inline src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al]: Is it better to be first pending and then with pragma also Removed in another version?

**Review by AndreasMoth — DISMISSED** [review]

File: src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al (hunk 2 above)
Comment (AndreasMoth) [reply src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al]: I believe this should be fine, it's an internal field that should not be used by anyone else.

Comment (AndreasMoth) [reply src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al]: This is no longer relevant as we now have capability of upgrading integer field to BigInteger.

**Review by AndreasMoth — DISMISSED** [review]

**Review by StefanSosic — APPROVED** [review]

**Review by darjoo — APPROVED** [review]

**Pull request merged**